db/data/20230309093808_fix_awarded_at_dates.rb

```diff
@@ -0,0 +1,337 @@
+      m4GbXNpiL23xW2j2TFvvEGdr
+      ZKu6jvfuZiaWmGFLQPv6DEtL
+      rKTKXah9jv7t42P5jqef7cGW
+      Fuc7cR6Yn3ro8GM9SwoZAysA
+    ]
+
+    awarded_at_dates = %w[
+      24/8/2022
+      24/8/2022
+      21/7/2022
+      12/1/2022
+      2/7/2022
+      23/9/2022
+      26/7/2022
+      8/9/2022
+      9/9/2022
+      9/9/2022
+      9/7/2022
+      31/8/2022
+      7/7/2022
+      7/9/2022
+      15/9/2022
+      1/9/2022
+      15/9/2022
+      15/9/2022
+      15/9/2022
+      15/9/2022
+      15/9/2022
+      15/9/2022
+      2/9/2022
+      15/9/2022
+      15/9/2022
+      15/9/2022
+      15/9/2022
+      24/8/2022
+      14/9/2022
+      16/9/2022
+      2/9/2022
+      2/9/2022
+      2/9/2022
+      2/9/2022
+      15/9/2022
+      15/9/2022
+      15/9/2022
+      10/8/2022
+      19/8/2022
+      10/8/2022
+      6/9/2022
+      6/9/2022
+      6/9/2022
+      6/9/2022
+      6/9/2022
+      6/9/2022
+      6/9/2022
+      25/8/2022
+      6/9/2021
+      25/8/2022
+      25/8/2022
+      25/8/2022
+      14/9/2022
+      25/8/2022
+      25/8/2022
+      5/9/2022
+      25/8/2022
+      25/8/2022
+      18/8/2022
+      25/8/2022
+      4/7/2022
+      23/7/2022
+      22/7/2022
+      20/7/2022
+      19/8/2022
+      19/8/2022
+      19/8/2022
+      19/8/2022
+      10/8/2022
+      19/8/2022
+      19/8/2022
+      10/8/2022
+      19/8/2022
+      19/8/2022
+      19/8/2022
+      10/8/2022
+      10/8/2022
+      19/8/2022
+      10/8/2022
+      10/8/2022
+      10/8/2022
+      19/8/2022
+      19/8/2022
+      19/8/2022
+      19/8/2022
+      10/8/2022
+      10/8/2022
+      2/9/2022
+      10/8/2022
+      19/8/2022
+      19/8/2022
+      30/8/2022
+      30/8/2022
+      30/8/2022
+      30/8/2022
+      30/8/2022
+      30/8/2022
+      30/8/2022
+      30/8/2022
+      30/8/2022
+      30/8/2022
+      24/10/2022
+      24/10/2022
+      24/10/2022
+      24/10/2022
+      24/10/2022
+      24/10/2022
+      24/10/2022
+      3/9/2022
+      4/7/2022
+      10/8/2022
+      10/8/2022
+      30/8/2022
+      30/8/2022
+      8/9/2022
+      8/9/2022
+      8/9/2022
+      8/9/2022
+      8/9/2022
+      8/9/2022
+      6/9/2022
+      2/12/2021
+      2/9/2022
+      19/7/2022
+      7/7/2022
+      14/9/2022
+      21/9/2022
+      14/9/2022
+      14/9/2022
+      14/9/2022
+      14/9/2022
+      14/9/2022
+      14/9/2022
+      14/9/2022
+      26/8/2022
+      24/8/2022
+      9/7/2022
+      7/7/2022
+      3/10/2022
+      5/4/2022
+      13/4/2021
+      28/7/2022
+      29/6/2022
+      1/2/2022
+      8/7/2022
+      8/7/2022
+      3/10/2022
+      31/8/2022
+      6/6/2022
+      10/07/2019
+      23/9/2022
+      11/03/2022
+      6/4/2022
+      15/9/2022
+      25/8/2022
+      20/7/2022
+      9/7/2022
+      15/9/2022
+      8/9/2022
+    ]
+
+    slugs.each_with_index do |slug, index|
+      Trainee.find_by(slug: slug).update(awarded_at: DateTime.strptime(awarded_at_dates[index], "%-d/%m/%Y"))
```

Comment:
Other environments won't have these trainees, so we need to protect against `nil` values using `&`:

```suggestion
      Trainee.find_by(slug: slug)&.update(awarded_at: awarded_at_dates[index])
```

Rails can handle date strings.

It's not a big deal, but you could simplify the code using a hash. Example:

```
{
  "VnxacCGTGfM9i65n4tn2tnMi" => "24/8/2022",
  "mQ7YMn9Hoq445p9RkmXTiRqc" => "24/8/2022",
  "b2wW14GLy2pghzhkFosPVwoQ" => "21/7/2022",
  "qnAk6TgyeBgRhEM1GxaDFiUX" => "12/1/2022",
  "mBmCkoh6mCa7d4yNo1TgUvCW" => "2/7/2022",
  "i4h1DvfSmXXisdwH66UnRXdm" => "23/9/2022",
  "e4oz3gFNJSpH63zwrjgGmfKQ" => "26/7/2022",
  "8Vkdr9U8WtcsRNU1aAWZMe73" => "8/9/2022",
  "uyMuY1TZ6N282FVo5piiih88" => "9/9/2022",
  "UByk9jiqEq5FBGqs3WViKDYE" => "9/9/2022",
  "h28h6GQy6kcaE7JBMGDe6DYs" => "9/7/2022",
  "VmGLqA8fTYjdmW4opU9yHXu2" => "31/8/2022",
  "PK1ApvFrzdm8wgVmThUs1huV" => "7/7/2022",
  "jUeJTU2qZ26ffSn4128PWKxp" => "7/9/2022",
  "8aaJCGREXkudHNEiTxSnT6sy" => "15/9/2022",
  "pSxJKR18AAAuWfaUbK4GkuqE" => "1/9/2022",
  "mg5Tq4sBxqXBVg4M5zbrwW6d" => "15/9/2022",
  "yPqkYKzmCp1Um96uHM8VaYzf" => "15/9/2022",
  "oHduNUAYHJNVhPet6VLVXNET" => "15/9/2022",
  "iSQmydYJFLfXkZjQ8eTm5W2y" => "15/9/2022",
  "Ugkk6fof7ijMEpMvehS74aiz" => "15/9/2022",
  "D4GNFj4oxauVTPya9fTervMp" => "15/9/2022",
  "EEUhF2vdw1XLTWV3sPoVVt9v" => "2/9/2022",
  "XFY5b2Cynu5NmeR1HTowmCzJ" => "15/9/2022",
  "UETeVELzBx8r5WYC1HCUr8Kj" => "15/9/2022",
  "M3NJWGSqZe1HFMs3gyGipig6" => "15/9/2022",
  "8Yaeq4RDDZ2ZZLNUyoqbzWr3" => "15/9/2022",
  "XpjtpGpkFtYXypop17cve2GA" => "24/8/2022",
  "FZjvEZWsgnMKYYSn59sJHpmT" => "14/9/2022",
  "Wd6bvBss5kGQkM83jzMihk3T" => "16/9/2022",
  "ZW55uE8kfQSGS8CDauNEZbde" => "2/9/2022",
  "f2LuC5UGxwAx4KxMynWccf6z" => "2/9/2022",
  "DCrAdQgftwZyaPucdaeKs7z9" => "2/9/2022",
  "CCjZ24YTGvnPYGfANwdoG9ho" => "2/9/2022",
  "hkKGzoUZEhZ7wdF7awJL7cEq" => "15/9/2022",
  "KJtUEWkKzTdSgGZVj5H4HwKy" => "15/9/2022",
  "B6XWtRZoys1bWtpYd2ZS36zh" => "15/9/2022",
  "K7DTHHfKNFrfdwFCnTxWV7qj" => "10/8/2022",
  "mDbaVo1FfmsBSHVoNZCgmfp6" => "19/8/2022",
  "ZyBVVBzHRZ9xCxKXk2a9QK2p" => "10/8/2022",
  "PCRGB7EJpJVmeGuRqRQfsacj" => "6/9/2022",
  "VRqzxpwKtadCtMfWxqLo18WW" => "6/9/2022",
  "SX4MBeBgmxUtBVZnYnMjaWXF" => "6/9/2022",
  "d2LAqvpRVQRJ7r1pxbjsyeno" => "6/9/2022",
  "ZCj4ScBAi6sD54Ayw5rRZhu1" => "6/9/2022",
  "J3vAJvWQTHZWcBX45pZaygjZ" => "6/9/2022",
  "eo9XjAfX9xGgF7X25beNtnvj" => "6/9/2022",
  "3QWKGvaxhGSLJaN6EB4pfzzs" => "25/8/2022",
  "J8ziVNUomJdxpt51x2pc8Wp9" => "6/9/2021",
  "NxexhYYyHQks7HqTnEeDsB55" => "25/8/2022",
  "DSriEGzcnrezomFck2bsG3gt" => "25/8/2022",
  "LjaEbYvFQ7wgRGjD8b7FLeHo" => "25/8/2022",
  "YbDnGVPauwXTEQBcV2znKHrH" => "14/9/2022",
  "gHS9azaW1SYuAYywZnWnAV8g" => "25/8/2022",
  "mfG6GAHpu3wEw4BUY5ofQVCt" => "25/8/2022",
  "4axeYPUY9eVV3kGt2YzChDu3" => "5/9/2022",
  "TGfCQvUFkBeTwNxGdhPFnkwh" => "25/8/2022",
  "3pk9ciddQeJTvvCgExgVBfJh" => "25/8/2022",
  "k6iDFnap9E98XtUkz1kLcqqs" => "18/8/2022",
  "aGCU8EQoJFnRx7EbASc2eTJM" => "25/8/2022",
  "5XyMiWLva3H7tFppZ8rqbxow" => "4/7/2022",
  "EcEfvq5AnbkQSvbqBA66W2kh" => "23/7/2022",
  "L21fFkVC4msLXSg61uoAVpXS" => "22/7/2022",
  "EmxaNnpRZs36nEWytmosBrbZ" => "20/7/2022",
  "vCBJQ7VewSrW23rpUEtDYnbw" => "19/8/2022",
  "7HqRLVDDfQSjdkgNypquSRxV" => "19/8/2022",
  "dd3sgqaSDpTyehRyqp1ZJ1zn" => "19/8/2022",
  "qmpoHZBGVMWq1bDZcFk3qNxm" => "19/8/2022",
  "pMDEZyhth5eJcEoQDUmfPMCD" => "10/8/2022",
  "FP1TuZHyhfyucHHRscrSwj1F" => "19/8/2022",
  "i2UpxWeAPGVAtnviXmzLgBM1" => "19/8/2022",
  "PW4AT38hEX35qRcAoDcCpt4J" => "10/8/2022",
  "QduoQMG5o9KpSpWdwmt5SyLe" => "19/8/2022",
  "QFA8KNUn1QAksNnYkpjo1u5D" => "19/8/2022",
  "E3tVVscZATwZrPF4q7tnSoX7" => "19/8/2022",
  "UCLq95eJaH5vw1KbgGWkWyqT" => "10/8/2022",
  "A3EgJrejAA7hh445RwEtZHZC" => "10/8/2022",
  "PxQTB3zZJGZE6vjnL8MUirZK" => "19/8/2022",
  "A9NoRrFFBbbYH6y7UCA3apzo" => "10/8/2022",
  "q2TMiqcBsFdUQEbVGMFpqvcf" => "10/8/2022",
  "Dgsi7ZXWxLRbuipDUTg8WZr9" => "10/8/2022",
  "9TYB9cEBfGQr3rx14pZ3vLKQ" => "19/8/2022",
  "NqJLsaAzCR8jw3qSUeyj832Z" => "19/8/2022",
  "gUyABjzpnqjfc6JhwxbwyExg" => "19/8/2022",
  "NCPHzS2Hs7B5mfsRWHAxYabe" => "19/8/2022",
  "nJJFJfzhtjY4ocZ3XJBxwQeT" => "10/8/2022",
  "zKHkmRJCrcUSA4jZhPZCn3YY" => "10/8/2022",
  "svn6P7soJ3JQc95CUJ7ZAWmv" => "2/9/2022",
  "A31jgfbMWZuCx3TCi54ekLuT" => "10/8/2022",
  "329q3WpXqrZfPQDThCMT7t1w" => "19/8/2022",
  "5mTTrcNdmoQ5RqHNNSomvp7r" => "19/8/2022",
  "hWjxLMuR9LFMr5zmWFZcigxS" => "30/8/2022",
  "4Bg7j3ZqKLvhRHHaR2Q3sskA" => "30/8/2022",
  "i1FuiqFe6e5QxXyEp8gDmpiS" => "30/8/2022",
  "XtgMZLQW9dL6Ls2V5x2GCMgz" => "30/8/2022",
  "f6PLsmbus6oiCUPpCzyaDifz" => "30/8/2022",
  "nxySUYYkZdccpEQ2TxtVYD6W" => "30/8/2022",
  "MCF4hQgjKtFQTkMxNzzpF7kg" => "30/8/2022",
  "ywfjUg8DnupjpdbdfsxBsh5f" => "30/8/2022",
  "LiFaA7ZeZzstzW6AaNEBV6DE" => "30/8/2022",
  "wLkHaUrJBDR8nUtCq8huEtNG" => "30/8/2022",
  "GJm8urEyAPssA57kDvtutr68" => " 4/10/2022",
  "AkZwM2HetGkCYuSAAPx7Mvpi" => " 4/10/2022",
  "RaLD1JuFPbzzq5r7BRJjmPoN" => " 4/10/2022",
  "qefMmYNFkD2fpp2YrAmw7mBw" => " 4/10/2022",
  "1nwqDQFKXKbviaGC9jKutonM" => " 4/10/2022",
  "JvnJZPZ28n1XtUupbVCF8eEo" => " 4/10/2022",
  "wPZZqKypCAVZDvSDiShipXph" => " 4/10/2022",
  "g3PYm1826GsZZJ9dxe7ovBmX" => "3/9/2022",
  "xScpuKfi8viVSd9LYUsQYFd8" => "4/7/2022",
  "MXUqPSGV9Wuuu2h8oD7rLqjN" => "10/8/2022",
  "4XHZVgSMP5uWPx4bvhoKjWPs" => "10/8/2022",
  "23haSuhD9qyPdChPHVULiZ5r" => "30/8/2022",
  "cLyrsK1hAQPdoc4ZPm9tRQ1d" => "30/8/2022",
  "bxtmJXb7UYp38Lgbcd6ZahsL" => "8/9/2022",
  "ngVXD4mtVRavPMgTWqHT4CpJ" => "8/9/2022",
  "rZ5Sm8DVN99sr6wqEqqsCqjc" => "8/9/2022",
  "B5cKeU8AFSgN7EbBAkzcCMG7" => "8/9/2022",
  "woh6cDfCDwi7LF4P7fPAWQi5" => "8/9/2022",
  "HnhfLionoSrvmwWDsbmHK6hk" => "8/9/2022",
  "mTrrpqoRSuKS8u3Bqr34TBai" => "6/9/2022",
  "5genr4MesiPhM1yoN9Ekpkk6" => "2/12/2021",
  "Uj6wusEzxjhRgEr6Mek56Zy8" => "2/9/2022",
  "4iScRWyNQUxE3tWVXBwr23Eq" => "19/7/2022",
  "t9LxogYd5xpx2j13hi6mnqtq" => "7/7/2022",
  "njweTKjKSfRYBfh2jypM3a31" => "14/9/2022",
  "wnip2ha6aBhngBhhXGaRZ2Yj" => "21/9/2022",
  "PQ7gbww7oASrjA7XEQdfhwzk" => "14/9/2022",
  "azkMGUbm5vpXUbSuhGC9Txhj" => "14/9/2022",
  "m5k8gFSTH9UEuyNW3gd3SP21" => "14/9/2022",
  "tNJbQgVzgeYdaBZvoisUtArL" => "14/9/2022",
  "rrm2MaCSp6eKwtKfdE6CqS27" => "14/9/2022",
  "NkvZLepNniM5DqKjUMfYb56h" => "14/9/2022",
  "ntqrEAFPAwYwdHYYHqik7XE5" => "14/9/2022",
  "Eevwnqmu3wmTfRcDn8kFxJ6C" => "26/8/2022",
  "XiYGDaZKJJUMpbwNaCFSQRaJ" => "24/8/2022",
  "5qMDJXiK6zLN4dadUdkwSpSm" => "9/7/2022",
  "pqt586r97PGj4KSde4trbtKo" => "7/7/2022",
  "HBo1J84SrzDHCArUuMQfPv8M" => "3/10/2022",
  "v36gUcGPYEnMVwWrCpHqPk1U" => "5/4/2022",
  "AMdvRUdAvf4hJsHYH8oeRaD5" => "13/4/2021",
  "C2w3NnAQCJShaqS2nZAdhy7z" => "28/7/2022",
  "9dNrq1vwhgW83BZe5uXejJrk" => "29/6/2022",
  "pmsQL4Y7H56Ts88bpd12VHoZ" => "1/2/2022",
  "nnpeb6yJuGaTtHw4DKQinhui" => "8/7/2022",
  "GfhEyNm1aRUNT6TvUEkN3XBC" => "8/7/2022",
  "QeQq5PxPLnwGaaxEZuKS8UX6" => "3/10/2022",
  "ZgKqYqVkipfLVMUCAWATAUeA" => "31/8/2022",
  "svrfpmJMBd9NcvanfW637SbZ" => "6/6/2022",
  "6Lzq7uNgiRHhMSHMBPBnZdTf" => "10/07/2019",
  "9EiAWZvPANi32aafX4bXPAhK" => "23/9/2022",
  "kyt5FyJF9JiYgukJ63r9pfuR" => "1/03/2022",
  "iC5ckjHpnm5WRKo6uvtADDff" => "6/4/2022",
  "NYfJSuyLZ1qB83RLpZdnncYL" => "15/9/2022",
  "NNjJaCDa4nTNoNWSuyND21WD" => "25/8/2022",
  "m4GbXNpiL23xW2j2TFvvEGdr" => "20/7/2022",
  "ZKu6jvfuZiaWmGFLQPv6DEtL" => "9/7/2022",
  "rKTKXah9jv7t42P5jqef7cGW" => "15/9/2022",
  "Fuc7cR6Yn3ro8GM9SwoZAysA" => "8/9/2022",
}.each do |slug, award_date|
  Trainee.find_by(slug: slug)&.update(awarded_at: award_date)
end
```